Makefile

```diff
@@ -61,10 +62,10 @@ help: ## Display this help.
 ##@ Build
 
 docker-build: ## Build docker image with the manager.
-	mvn clean package -Dquarkus.container-image.build=true -Dquarkus.container-image.image=${IMG}
+	mvn clean package -Pnative -Dquarkus.container-image.build=true -Dquarkus.container-image.image=${IMG} -Dquarkus.native.container-runtime=${IMAGE_BUILDER}
```

Comment:
is'nt it `-Dnative`

Reply:
doesn't work.  it's -P

Reply:
worked for me .. even though -P is for profile and -D is for properties 🤔

Reply:
When you pass -D and try setting the builder to podman, it still looks for docker.
Looks like only -P activates the container-runtime setting used later in this line.